resources/views/masjid_profile/show.blade.php

```diff
@@ -11,6 +11,13 @@
         <div class="card">
             <table class="table table-sm card-table">
                 <tbody>
+                    <tr>
+                        <td colspan="2" class="text-center">
+                        @if (Setting::get('masjid_logo_path'))
+                            <img class="img-fluid my-4" src="{{ Storage::url(Setting::get('masjid_logo_path'))}}" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
```

Comment:
Di fitur setting itu kita bisa kasih default value mas: `Setting::get('key', 'default_value');`. Jadi default valuenya ambil nama masjid dari config saja.

```suggestion
                            <img class="img-fluid my-4" src="{{ Storage::url(Setting::get('masjid_logo_path'))}}" alt="{{ Setting::get('masjid_name', config('masjid.name')) }}">
```